src/App.js

```diff
@@ -25,7 +25,7 @@ export default function App() {
         <BrowserRouter>
```

Comment:
your css and layout needs a little more work, just making the card look distinct and adding the right grid/flex is fine.

Your notification buttons are a little jarring, esp as we haven't implemented any notification type stuff in the back end. Be wary of adding things that look like features if we can't do them :)